crates/bevy_pbr/src/render/mesh_view_bindings.rs

```diff
@@ -484,6 +489,7 @@ pub fn prepare_mesh_view_bind_groups(
         Some(light_probes_binding),
         Some(visibility_ranges_buffer),
         Some(ssr_binding),
+        Some(environment_map_binding),
```

Comment:
Will exit if no `EnvironmentMapLight ` is present in the scene, need to implement a fallback uniform for the `environment_map_binding`